File: src/main/java/it/contrader/dto/OggettoDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class OggettoDTO {
	
	private Long id;
	
	private String nome;
	
	private int dimensione;
	
	private boolean cancellato;
}

File: src/dto/oggettodto.ts
export class OggettoDTO {
    
    id: number;
    nome: string;
    dimensione: number;
    cancellato: boolean;
    cella: boolean;
    constructor(nome: string, dimensione: number) {
        this.nome = nome;
        this.dimensione = dimensione;
        this.cancellato = false;
        this.cella=false;
    }
}
